install.js

```diff
@@ -7,101 +7,127 @@ const tmp = require('tmp');
 
 const config = require('./config');
 const utils = require('./utils');
+const packageJson = require('./package');
 
 const CDN_URL = 'https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/';
+const BUILD_SEARCH_URL = 'https://omahaproxy.appspot.com/deps.json';
 
 function getOsCdnUrl() {
-    let url = CDN_URL;
+	let url = CDN_URL;
 
-    const platform = process.platform;
+	const platform = process.platform;
 
-    if (platform === 'linux') {
-        url += 'Linux';
-        if (process.arch === 'x64') {
-            url += '_x64';
-        }
-    } else if (platform === 'win32') {
-        url += 'Win';
-        if (process.arch === 'x64') {
-            url += '_x64';
-        }
-    } else if (platform === 'darwin') {
-        url += 'Mac';
-    } else {
-        console.log('Unknown platform or architecture found:', process.platform, process.arch);
-        throw new Error('Unsupported platform');
-    }
+	if (platform === 'linux') {
+		url += 'Linux';
+		if (process.arch === 'x64') {
+			url += '_x64';
+		}
+	} else if (platform === 'win32') {
+		url += 'Win';
+		if (process.arch === 'x64') {
+			url += '_x64';
+		}
+	} else if (platform === 'darwin') {
+		url += 'Mac';
+	} else {
+		console.log('Unknown platform or architecture found:', process.platform, process.arch);
+		throw new Error('Unsupported platform');
+	}
 
-    return url;
+	return url;
 }
 
-function getLatestRevisionNumber() {
-    return new Promise((resolve, reject) => {
-        const url = getOsCdnUrl() + '%2FLAST_CHANGE?alt=media';
-        got(url)
-            .then(response => {
-                resolve(response.body);
-            })
-            .catch(err => {
-                console.log('An error occured while trying to retrieve latest revision number', err);
-                reject(err);
-            });
-    });
+function getBuildNumberByVersion() {
+	return new Promise((resolve, reject) => {
+		got(
+			BUILD_SEARCH_URL,
+			{
+				query: {
+					version: packageJson.version
```

Comment:
package.json contains semver "version" field which is allows MAJOR.MINOR.PATCH, for digits will be not valid according to semver. Also some part of version should be for package related fixes.